src/GeositeFramework/plugins/map_utils/export/main.js

```diff
@@ -0,0 +1,20 @@
+define([
+    'dojo/_base/declare'
+    ],
+    function(declare) {
+    'use strict';
+
+    return declare(null, {
+        constructor: function(args) {
+            declare.safeMixin(this, args);
+        },
+
+        execute: function() {
+            this.app.dispatcher.trigger('export-map:pane-0');
+        },
+
+        cancel: function() {
+            // NOP
```

Comment:
It's a stand-in for "no operation" but maybe I'll just delete these placeholder functions instead.